feature/appdetail/src/main/kotlin/com/merxury/blocker/feature/appdetail/AppDetailViewModel.kt

```diff
@@ -106,6 +106,7 @@ import kotlinx.coroutines.flow.onStart
 import kotlinx.coroutines.flow.update
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
+import org.jetbrains.annotations.VisibleForTesting
```

Comment:
I think you should import `androidx.annotation.VisibleForTesting` instead of `org.jetbrains.annotations.VisibleForTesting`